src/template_part.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os
+
+
+class TemplatePart:
+    """
+    Class to manage a template part for ElabFTW.
+    """
+
+    def __init__(self, template_part_file):
+        """
+        Initializes the TemplatePart object.
+
+        Parameters:
+            template_part_file (str): Path to the JSON file.
+        """
+        self.template_part_file = template_part_file
+        self.template_part = self.read_template_part(template_part_file)
```

Comment:
- please define template_part in the attributes of the class (for now, it is not defined)
- also, one question: I think a TemplatePart object should be able to exist without a corresponding file (think about the unit test that you will implement), so the constructor should be restructured (template_part_file should not be mandatory to create an object of this class)